src/app/work-item/work-item-detail/markdown-control/markdown-control.component.html

```diff
@@ -54,6 +54,7 @@
   <div class="col-md-12 col-sm-12 col-xs-12 detail-text-action">
       <span class="col-md-4">
         <small *ngIf="loggedIn && tabBarVisible">
+          <img src="../../../../assets/img/markdown-solid.png" class="markdown-mark" />
           <b>Markdown Supported</b>
```

Comment:
I think we need to go one more level to find the png?  
I.e.
`../../../../../assets/img/markdown-solid.png`
rather than 
`../../../../assets/img/markdown-solid.png`